### ISSUE TITLE
Reflect disabled post type options in settings.

### ISSUE THREAD

**Issue by sybrew** [sybrew/the-seo-framework#488]
When you disable a post type, it isn't clear that any other settings for the post type will no longer be active. This leads to confusion and even false assumptions on whether a broken post type should be indexed or not.

So, we'd want to (de)highlight those settings for disabled post types, or find another way to annotate it, for example, via a tooltip.

![image](https://user-images.githubusercontent.com/7662236/69223655-e0493b00-0b7b-11ea-8a9b-e49ad043a1c2.png)
